packages/shared/lib/db/migrations/20240118110056_add_custom_column.cjs

```diff
@@ -0,0 +1,13 @@
+const tableName = 'nango._nango_configs';
```

Comment:
Todo: is this migration different from the others? Why is it needed to prefix the table?